### DISCUSSION TITLE
Dropout

### DISCUSSION THREAD

**Discussion by tortijazz** [netket/netket#1695] [Q&A]
Hi, everybody

Quick question: Is there a way to make an RBM sparse inside Netket, which is known as "dropout" in Artificial Neural Networks?

Thanks.

**Comment by PhilipVinc** [comment]
Out of the box it won't work because we are not passing the arguments that signal that you are training/inferencing to the network.

You need to build on top of this PR https://github.com/netket/netket/pull/1485 (note that the author there wanted to use dropout)

I think the PR is 80% there. I'm unsure what is missing. Possibly support in QGT for using SR.

To get it merged only some tests are missing, otherwise it would be good to go.
